quarkchain/cluster/grpc_client.py

```diff
@@ -46,10 +47,16 @@ def add_root_block(self, root_block: RootBlock) -> bool:
         )
         try:
             response = self.client.AddRootBlock(request)
-        except Exception:
+        except grpc.RpcError as e:
+            Logger.log_exception()
             return False
 
         if response.status.code == 0:
             return True
         else:
+            Logger.warning(
```

Comment:
consider `Logger.error`

also, a nit suggestion on coding styles (inspired from golang) that is we can handle errors first, then proceed to normal case. so I would suggest following:

```
if response.status.code != 0:
  # warning and return False

return True
```

Reply:
Changed.